README.md

```diff
@@ -1,35 +1,57 @@
+
+### Room/channel permissions linkage
+
+When you create a Hubs room using the `!hubs create` bot command, you establish a permanent association between the Hubs room and the Discord channel where you typed the command. This association will cause the Hubs room to use information from your Discord server to authenticate participants. Specifically:
+
+- People can only join the Hubs room via Discord OAuth.
+- When they join, their permissions are based on their Discord roles. (People with Discord "manage channels" permission will be able to change the name and scene in the room, and people with Discord "kick users" permission will be able to kick people.)
+- Their display name in the Hubs room will reflect their Discord display name.
+
+This only happens with rooms that you create using `!hubs create` -- simply bridging a room by putting it in the topic won't cause it to become permission-linked. This linkage will persist for the lifetime of the Hubs room -- if you don't like it, make a new Hubs room.
 
-The primary function of the bot is to establish a 1:1 linkage between a Discord text channel and a Hubs room. Once rooms are linked, then the bot will do the following:
+### Room/channel bridging
 
-- Bridge text chat from the Discord channel into the Hubs room.
-- Bridge text chat from the Hubs room into the Discord channel.
-- Post links in Discord to media (images, videos, models) which are pinned in the Hubs room.
-- Post in the Discord channel when someone joins or leaves the Hubs room, or if administrative stuff happens in the Hubs room.
+Independently of being permission-linked, the bot will detect any Hubs rooms in channel topics in channels that the bot can read and join those rooms, establishing a bridge between the room and the Discord channel. Specifically:
 
-When a Hubs room is associated with a Discord channel, users will be assigned abilities in the Hubs room based on their Discord roles. For example, Discord owners and moderators will be able to change settings on a Hubs room and be able to moderate users in the room.
+- A notification will appear in the Discord channel when someone joins or leaves the Hubs room, or if administrative stuff happens in the Hubs room.
+- Text chat and images will be bridged from the Discord channel into the Hubs room.
+- Text chat and photos will be bridged from the Hubs room into the Discord channel.
+- Links to media (images, videos, models) which are *pinned* in the Hubs room will be bridged to Discord.
+
+Note that you need to set up a webhook for the bot to use in the Discord channel, or it won't be able to post chat from Hubs.
+
+If you remove the Hubs room from the topic, bridging will stop.
 
 ### Great. I want to run this on my Discord server.
 
-1. Request an [invite link][bot-invite] to invite the bot to your guild by emailing hubs@mozilla.com.
+The bot is currently in a sort of closed beta, so you'll need to request an invite by emailing [hubs@mozilla.com][bot-invite].
```

Comment:
```suggestion
The bot is currently in a closed beta, so you'll need to request an invite by emailing [hubs@mozilla.com][bot-invite].
```
A smidge less confusing if you remove the qualifier

Reply:
The truth hurts!